src/index.test.ts

```diff
@@ -501,6 +502,30 @@ describe("getSimulationComputeUnits", () => {
   });
 });
 
+describe("makeTokenMint", () => {
+  test("makeTokenMint works", async () => {
+    const mintAuthority = Keypair.generate();
+    const connection = new Connection(LOCALHOST);
+    await airdropIfRequired(
+      connection,
+      mintAuthority.publicKey,
+      100 * LAMPORTS_PER_SOL,
+      1 * LAMPORTS_PER_SOL,
+    );
+
+    const signature = await makeTokenMint(
+      connection,
+      mintAuthority,
+      "Unit test token",
+      "TEST",
+      9,
+      "https://example.com",
+    );
```

Comment:
Test should also verify the metadata passed into helper is successfully stored on the mint after transaction is sent.

For example, the `makeTokenMint` helper function accepts a `additionalMetadata` parameter, but doesn't include the `createUpdateFieldInstruction` instruction to add the custom metadata.

Reply:
Done!